rplidar.go

```diff
@@ -69,6 +76,15 @@ func newRplidar(ctx context.Context, _ resource.Dependencies, c resource.Config,
 		return nil, err
 	}
 
+	rplidarModel := svcConf.RplidarModel
+	if rplidarModel == "" {
+		logger.Info("No rplidar model given, setting to default of A1")
+		rplidarModel = "A1"
```

Comment:
Maybe better to require the user to specify the model? Will this break a bunch of our example configs? Do we need to make docs changes? Should this go in the scope doc?

Reply:
Added to scope